docs/pages/_app.js

```diff
@@ -88,11 +88,7 @@ function forcePageReload(registration) {
 }
 
 async function registerServiceWorker() {
-  if (
-    'serviceWorker' in navigator &&
-    process.env.NODE_ENV === 'production' &&
-    window.location.host.indexOf('mui.com') !== -1
-  ) {
+  if ('serviceWorker' in navigator && process.env.NODE_ENV === 'production') {
```

Comment:
This change is here only for demonstration purposes in deploy preview, I'll revert it before merging